src/main.js

```diff
@@ -37,6 +37,7 @@ require.config({
         // implementations (e.g. cloud-based storage).
         "fileSystemImpl"    : "filesystem/impls/filer/FilerFileSystem",
 
+        "caman"             : "thirdparty/caman/caman.full.min",
```

Comment:
@humphd we would need to deal with load timing issues in that case. One thing I would like to do instead (in a follow-up issue) is to remove a bunch of the cruft in Caman.js that we don't use, i.e. create a customized built file and use that instead. I bet we can reduce the size significantly with that.